src/hooks/useDrag.tsx

```diff
@@ -184,6 +184,31 @@ export const useDrag: OnDragEndResponseWrapper = (
   return updateHomepageState(result, homepageState)
```

Comment:
this PR mainly changes the condiitonal logic from switch-case to if-else right? Why does switch-case not support
`<subitem name>-<number>`?

Reply:
^switch case not as powerful as `match` sadly enough :/